### PR TITLE
redesigned pages and category model

### PR DESCRIPTION
**Pull request by muharremunver**
Please review the code. I will add something tomorrow

### CODE REVIEW

File: backend/eureka/eureka/settings.py [hunk 1]
```diff
@@ -41,7 +41,7 @@
     'post',
     'home',
     'api',
-
+    'avatar',
```
Comment (onatbas) [inline backend/eureka/eureka/settings.py]: Why is this required?

File: backend/eureka/post/models.py [hunk 2]
```diff
@@ -1,18 +1,57 @@
...
+
     class Meta:
         ordering = ['-publishing_date']
+
+
+class Comment(models.Model):
+    post = models.ForeignKey('post.Post', related_name='comments',on_delete=models.CASCADE)
+    name = models.CharField(max_length=200,verbose_name='Username')
+    content = models.TextField(verbose_name='Comment')
+    created_Date = models.DateTimeField(auto_now_add=True)
```
Comment (onatbas) [inline backend/eureka/post/models.py]: Commenting is not something we support. Can we remove this, please? Comments are going to be in form of annotations.

File: backend/eureka/post/views.py [hunk 3]
```diff
@@ -1,6 +1,6 @@
 from django.shortcuts import render, HttpResponse, get_object_or_404, HttpResponseRedirect, redirect, Http404
 from .models import Post
-from .form import PostForm
+from .form import PostForm, CommentForm
```
Comment (onatbas) [inline backend/eureka/post/views.py]: Please remove CommentForm

File: backend/eureka/templates/post/detail.html [hunk 4]
```diff
@@ -5,36 +5,36 @@
...
+                  <br><br><hr>
+                  <h3>Add comment</h3>
+                  {% include 'post/comment.html' %}
+                  <hr>
+
+                  {% for comment in post.comments.all %}
+                    <h4>{{ comment.name }} | <small>{{ comment.created_Date|timesince }} before</small></h4>
+                    <p>{{ comment.content|linebreaks }}</p>
+                  <hr>
+                  {% endfor %}
```
Comment (onatbas) [inline backend/eureka/templates/post/detail.html]: Please remove any comment related features from the app. Comments are not a feature we support.

**Comment by muharremunver** [comment]
Thanks for your review. 
1. I had used avatar package to add avatar to users. As we spoke, you have more useful solution. Let's use it.
ckEditor('RichTextField') is used for content field of posts.  We can remove avatar package, what should we use instead of this.
2. comment model deleted!
3. Okay, I will try as you said in my next changesets.

**Comment by onatbas** [comment]
Looks OK to me. Post page is not working now , but let's have a different PR for that.
